st2api/tests/unit/controllers/v1/test_packs_views.py

```diff
@@ -133,7 +133,7 @@ def test_get_pack_files_and_pack_file_ref_doesnt_equal_pack_name(self):
         # Ref is not equal to the name, controller should still work
         resp = self.app.get("/v1/packs/views/files/dummy_pack_16")
         self.assertEqual(resp.status_int, http_client.OK)
-        self.assertEqual(len(resp.json), 3)
+        self.assertEqual(len(resp.json), 4)
```

Comment:
I had to bump the file count in the pack because this PR added a `BUILD` file in the pack.